tests/unit/parsers/test.manifestjson.js

```diff
@@ -102,6 +102,22 @@ describe('ManifestJSONParser', () => {
       });
     });
 
+    it('should fail on id longer than 255 characters', () => {
+      const addonLinter = new Linter({ _: ['bar'] });
+      const json = validManifestJSON({
+        applications: { gecko: { id: `@${'a'.repeat(255)}` } }, // 256 chars
```

Comment:
Mind documenting here, that we don't have to care about unicode validation as this is handled by the schema as well?